internal/logger/logger.go

```diff
@@ -0,0 +1,45 @@
+package logger
+
+import (
+	"io"
+	"log/slog"
+	"os"
+
+	"github.com/dicedb/dice/config"
+	"github.com/rs/zerolog"
+)
+
+func getLogLevel() slog.Leveler {
+	var level slog.Leveler
+	switch config.DiceConfig.Server.LogLevel {
+	case "debug":
+		level = slog.LevelDebug
+	case "info":
+		level = slog.LevelInfo
```

Comment:
Same as above, remove `case "info"` as default will take care of it to reduce code redundancy.